integrations/github-app/syncs/pull-requests.ts

```
@@ -0,0 +1,91 @@
+import type { NangoSync, ProxyConfiguration, GithubMetadataInput, GithubPullRequest } from '../models.js';
+import { DEFAULT_SYNC_WINDOW, RETRIES } from '../constants.js';
+import { getPullRequestsQuery } from '../graphql/pull-requests.js';
+import { toPullRequest } from '../mappers/to-pull-request.js';
+import { githubMetadataInputSchema } from '../schema.zod.js';
+import type { PullRequestQueryGraphQLResponse } from '../types';
+import { PullRequestState } from '../types';
```

ESLint

[error] Missing file extension for "../types"

(import/extensions)